### PR TITLE
bugfix - infinite loop in observe resize

### PR DESCRIPTION
**Pull request by kushagrms**
## Previous Behavior
There were some console errors in small width devices like iPhoneSE  `ResizeObserver loop limit exceeded`

## New Behavior
After wrapping around the function call with requestAnimationFrame, the errors are gone.

## Related Issue(s)

- Fixes # https://icmcdn.akamaized.net/imp/v3/incidents/details/367851747/home

### CODE REVIEW

**Comment by kushagrms** [comment]
> @kushagrms please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
>  > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement
> ### Contribution License Agreement
> This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> 
> 1. **Definitions**.
>    **“Code”** means the computer software code, whether in human-readable or machine-executable form,
>    that is delivered by You to Microsoft under this Agreement.
>    **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
>    approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
>    **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
>    Project, including but not limited to communication on electronic mailing lists, source code control
>    systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
>    discussing and improving that Project, but excluding communication that is conspicuously marked or
>    otherwise designated in writing by You as “Not a Submission.”
>    **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
>    associated comments and documentation.
> 2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
>    Project. This Agreement covers any and all Submissions that You, now or in the future (except as
>    described in Section 4 below), Submit to any Project.
> 3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
>    Should You wish to Submit materials that are not Your original work, You may Submit them separately
>    to the Project if You (a) retain all copyright and license information that was in the materials as You
>    received them, (b) in the description accompanying Your Submission, include the phrase “Submission
>    containing materials of a third party:” followed by the names of the third party and any licenses or other
>    restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
>    guidelines concerning Submissions.
> 4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
>    for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
>    Submission is made in the course of Your work for an employer or Your employer has intellectual
>    property rights in Your Submission by contract or applicable law, You must secure permission from Your
>    employer to make the Submission before signing this Agreement. In that case, the term “You” in this
>    Agreement will refer to You and the employer collectively. If You change employers in the future and
>    desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
>    and secure permission from the new employer before Submitting those Submissions.
> 5. **Licenses**.
> 
> * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
>   Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
>   the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
>   parties.
> * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
>   Your patent claims that are necessarily infringed by the Submission or the combination of the
>   Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
>   import or otherwise dispose of the Submission alone or with the Project.
> * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
>   No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
>   granted by implication, exhaustion, estoppel or otherwise.
> 
> 6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
>    licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
>    have disclosed under Section 3). You represent that You have secured permission from Your employer to
>    make the Submission in cases where Your Submission is made in the course of Your work for Your
>    employer or Your employer has intellectual property rights in Your Submission by contract or applicable
>    law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
>    have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
>    You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
>    REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
>    EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
>    PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
>    NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> 7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
>    You later become aware that would make Your representations in this Agreement inaccurate in any
>    respect.
> 8. **Information about Submissions**. You agree that contributions to Projects and information about
>    contributions may be maintained indefinitely and disclosed publicly, including Your name and other
>    information that You submit with Your Submission.
> 9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
>    the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
>    Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
>    exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
>    defenses of lack of personal jurisdiction and forum non-conveniens.
> 10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
>     supersedes any and all prior agreements, understandings or communications, written or oral, between
>     the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

@microsoft-github-policy-service agree company="Microsoft"

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5a8d7a29645b3b500eb7145d2e42ac34fcafc4c6'>5a8d7a29645b3b500eb7145d2e42ac34fcafc4c6</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/BaseButton-mount_base.html">986</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/BaseButton-mount.html">974</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Breadcrumb-mount_base.html">2355</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Breadcrumb-mount.html">2419</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Checkbox-mount_base.html">2249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Checkbox-mount.html">2243</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/CheckboxBase-mount_base.html">1989</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/CheckboxBase-mount.html">2007</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ChoiceGroup-mount_base.html">3893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ChoiceGroup-mount.html">3882</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ComboBox-mount_base.html">1079</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ComboBox-mount.html">1049</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/CommandBar-mount_base.html">8517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/CommandBar-mount.html">8593</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ContextualMenu-mount_base.html">18121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ContextualMenu-mount.html">18735</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DefaultButton-mount_base.html">1135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DefaultButton-mount.html">1154</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DetailsRow-mount_base.html">3062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DetailsRow-mount.html">3054</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DetailsRowFast-mount_base.html">3054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DetailsRowFast-mount.html">3097</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DetailsRowNoStyles-mount_base.html">2870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DetailsRowNoStyles-mount.html">2902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Dialog-mount_base.html">3399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Dialog-mount.html">3414</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DocumentCardTitle-mount_base.html">459</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/DocumentCardTitle-mount.html">456</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Dropdown-mount_base.html">2731</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Dropdown-mount.html">2742</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/FocusTrapZone-mount_base.html">1642</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/FocusTrapZone-mount.html">1668</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/FocusZone-mount_base.html">1604</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/FocusZone-mount.html">1616</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedList-mount_base.html">50566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedList-mount.html">59164</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedList-virtual-rerender_base.html">24505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedList-virtual-rerender.html">24528</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">75606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedList-virtual-rerender-with-unmount.html">75834</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedListV2-mount_base.html">440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedListV2-mount.html">452</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedListV2-virtual-rerender_base.html">415</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedListV2-virtual-rerender.html">422</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">447</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/IconButton-mount_base.html">1624</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/IconButton-mount.html">1595</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Label-mount_base.html">570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Label-mount.html">575</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Layer-mount_base.html">3740</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Layer-mount.html">3720</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Link-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Link-mount.html">638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/MenuButton-mount_base.html">1414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/MenuButton-mount.html">1388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/MessageBar-mount_base.html">27876</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/MessageBar-mount.html">27826</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Nav-mount_base.html">2738</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Nav-mount.html">2778</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/OverflowSet-mount_base.html">1115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/OverflowSet-mount.html">1145</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Panel-mount_base.html">2143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Panel-mount.html">2166</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Persona-mount_base.html">1078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Persona-mount.html">1087</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Pivot-mount_base.html">1343</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Pivot-mount.html">1336</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/PrimaryButton-mount_base.html">1236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/PrimaryButton-mount.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Rating-mount_base.html">5783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Rating-mount.html">5853</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SearchBox-mount_base.html">1270</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SearchBox-mount.html">1270</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Shimmer-mount_base.html">2453</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Shimmer-mount.html">2428</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Slider-mount_base.html">1816</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Slider-mount.html">1809</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SpinButton-mount_base.html">4182</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SpinButton-mount.html">3884</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Spinner-mount_base.html">644</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Spinner-mount.html">617</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SplitButton-mount_base.html">2610</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SplitButton-mount.html">2651</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Stack-mount_base.html">656</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Stack-mount.html">674</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/StackWithIntrinsicChildren-mount_base.html">1264</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/StackWithIntrinsicChildren-mount.html">1275</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/StackWithTextChildren-mount_base.html">3378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/StackWithTextChildren-mount.html">3390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SwatchColorPicker-mount_base.html">8317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/SwatchColorPicker-mount.html">8366</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/TagPicker-mount_base.html">2149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/TagPicker-mount.html">2178</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Text-mount_base.html">626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Text-mount.html">623</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/TextField-mount_base.html">1325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/TextField-mount.html">1328</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ThemeProvider-mount_base.html">1234</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ThemeProvider-mount.html">1234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ThemeProvider-virtual-rerender_base.html">887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ThemeProvider-virtual-rerender.html">880</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1824</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1829</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Toggle-mount_base.html">896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/Toggle-mount.html">912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/buttonNative-mount_base.html">411</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27216/perf-test/buttonNative-mount.html">412</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-ef436e96-9890-4ea2-826f-4b607a9087c4.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-ef436e96-9890-4ea2-826f-4b607a9087c4.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27216&commitId=9537eadc2bf7fa249d6f5e8529523339ad54a384&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by spmonahan** [comment]
Closing for housekeeping. If it is still relevant please re-open.

**Pull request closed without merging**